eclipse-che/che-server
# Fix vulnerable dependencies

### What does this PR do?
* Update vulnerable dependencies in order to fix the security issues according to the trivy report
* Update other dependencies according by merging dependabot pull requests.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/22062

### How to test this PR?

1. Git checkout to the PR branch.
2. Run `mvn clean install && cd dockerfiles && ./build.sh && cd .. && docker run aquasec/trivy image quay.io/eclipse/che-server:next`
3. See the report:

| Library  | Vulnerability  |  Severity | InstalledVersion  | FixedVersion | Title |
|---|---|---|---|---|---|
| com.h2database:h2 (api.war) | CVE-2022-45868 | HIGH | 2.1.210 |   |   |
| org.yaml:snakeyaml (api.war) | CVE-2022-1471 | CRITICAL | 1.33 | 2.0  |   |

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
